Autopilot/Inc/airspeed.hpp

```diff
@@ -55,6 +58,58 @@ class airspeed {
  * Derived classes
  **********************************************************************************************************************/
 
+class MPXV7002DP : public airspeed {
```

Comment:
Note for future, make all classes use upper CammelCase. 

If you could add this to the future tasks doc that would be amazing!